core/base/batch_multi_vector_kernels.hpp

```diff
@@ -6,13 +6,11 @@
 #define GKO_CORE_BASE_BATCH_MULTI_VECTOR_KERNELS_HPP_
 
 
```

Comment:
Change this to also a single newline ?

Reply:
This is not handled by clang-format. clang-format only changes the number of empty lines within the include declaration block. For everything before or after, the normal rule of max. 2 empty lines holds.